### PR TITLE
Rephrase text in PLPMTUD

### LINKED ISSUE #1335: A PADDING frame has no content.
PLPMTUD section contains the statement:

> The PADDING frame provides a useful option for PMTU probe packets that does not exist in other transports. PADDING frames generate acknowledgements, but their content need not be delivered reliably.

Elsewhere the document reiterates the fact that A PADDING frame has no content.

### PR DESCRIPTION
**Pull request by janaiyengar**
Closes #1335.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -3088,11 +3088,11 @@ actually lost.
 
 
 The PADDING frame provides a useful option for PMTU probe packets that does not
```
Comment (ianswett) [inline draft-ietf-quic-transport.md]: nit: Other transports don't have padding?

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -3088,11 +3088,11 @@ actually lost.
...
-exist in other transports. PADDING frames generate acknowledgements, but their
-content need not be delivered reliably. PADDING frames may delay the delivery of
-application data, as they consume the congestion window. However, by definition
-their likely loss in a probe packet does not require delay-inducing
-retransmission of application data.
+exist in other transports. PADDING frames generate acknowledgements, but they
+need not be delivered reliably. As a result, the likely loss of PADDING frames
```
Comment (ianswett) [inline draft-ietf-quic-transport.md]: nit: likely is speculative, I'd remove it.

File: draft-ietf-quic-transport.md (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-quic-transport.md]: Yeah, I'd drop the does not exist bit.  It's a big claim and not really necessary.

**Comment by LPardue** [comment]
LGTM as fix for the issue I raised

**Comment by janaiyengar** [comment]
I'll just merge this, since I have yours approvals.

**Pull request merged**